traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -271,11 +270,10 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	cdn := inf.Params["cdn"]
-
 	// We never store tm_path, even though low API versions show it in responses.
 	crConfig, err := Make(inf.Tx.Tx, cdn, inf.User.UserName, r.Host, inf.Config.Version, inf.Config.CRConfigUseRequestHost, false)
 	if err != nil {
-		writePerlHTMLErr(w, r, inf.Tx.Tx, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()), err)
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, err, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()))
```

Comment:
The error returned from `crconfig.Make()` is 500-level error and should not be displayed to the user. Right now, requesting `GET /tools/write_crconfig/nonexistent` yields:

```
HTTP/2 500
access-control-allow-credentials: true
access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
access-control-allow-methods: POST,GET,OPTIONS,PUT,DELETE
access-control-allow-origin: *
content-type: application/json
set-cookie: mojolicious=...; Path=/; Expires=Fri, 05 Mar 2021 22:48:36 GMT; Max-Age=3600; HttpOnly
vary: Accept-Encoding
whole-content-sha512: 5gzjE1ZKa484M1quoUVNTeNS4mMdzk0m/3uxQizzQ+hUmBaLWvmregVOjDa2I1cVeVRPX5Mw5KTym3iJxRUBhQ==
x-server-name: traffic_ops_golang/
content-length: 123
date: Fri, 05 Mar 2021 21:48:36 GMT

{"alerts":[{"text":"Error getting CDN info: Error querying CDN domain name: sql: no rows in result set","level":"error"}]}
```

Reply:
Fixed now.